src/components/crownConfig.vue

```diff
@@ -23,24 +23,26 @@
       :title="$t('Message Flow')"
     />
 
-    <div class="dropdown-container">
-      <button class="button" @click="dropdownOpen = !dropdownOpen">
+    <div class="cog-container" role="menubar">
+      <button class="cog-container--button" @click="dropdownOpen = !dropdownOpen">
         <i class="fas fa-cog"/>
       </button>
 
-      <ul class="dropdown" v-if="dropdownOpen">
-        <li>
+      <ul class="element-list" v-if="dropdownOpen" role="list">
```

Comment:
From https://www.w3.org/TR/html-aria:
> Note that some user agents suppress a list's implicit ARIA semantics if list markers are removed. Authors can use role=list to reinstate the role, if necessary.

So good call on specifying the role attribute here.